pootle/apps/pootle_fs/display.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from pootle_fs.response import FS_RESPONSE
+
+
+FS_EXISTS = [
+    "pulled_to_pootle", "pushed_to_fs", "fetched_from_fs",
+    "staged_for_merge_fs", "staged_for_merge_pootle",
+    "merged_from_fs", "merged_from_pootle"]
+
+STORE_EXISTS = [
+    "pulled_to_pootle", "pushed_to_fs", "added_from_pootle",
+    "staged_for_merge_fs", "staged_for_merge_pootle",
+    "merged_from_fs", "merged_from_pootle"]
```

Comment:
Just for simplifying maintaning this I would add a `BOTH_EXIST`, even if it is only used to then define `FS_EXISTS` and `STORE_EXISTS`:

```python
BOTH_EXIST = [
    "pulled_to_pootle",
    "pushed_to_fs",
    "staged_for_merge_fs",
    "staged_for_merge_pootle",
    "merged_from_fs",
    "merged_from_pootle",
]

FS_EXISTS = BOTH_EXIST + ["fetched_from_fs"]

STORE_EXISTS = BOTH_EXIST + ["added_from_pootle"]
```